logic/ball.py

```diff
@@ -28,7 +25,7 @@ def __init__(self, x: int, y: int, color: Color, radius: int = 10):
         self.color = color
         self.radius = radius
 
-    def dist(self, other: Self):
+    def dist(self, other):
```

Comment:
```suggestion
    def dist(self, other: Ball):
```
En important future ça fonctionne